### PR TITLE
Catalog done

### PR DESCRIPTION
**Pull request by illiarerun**
- [DEMO LINK](https://illiarerun.github.io/layout_catalog/)
- [TEST REPORT LINK](https://illiarerun.github.io/layout_catalog/report/html_report/)

I've used `margins` cause tests don't use `gap` property

### CODE REVIEW

**Comment by dima-tkachenko** [comment]
good work to my mind

File: src/styles/blocks/button-buy.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  font-weight: $weight-bold;
+  font-size: $m-text;
+  text-decoration: none;
+  color: inherit;
+}
+
+.button-buy:hover {
+  background-color: #fff;
+  color: $blue-accent;
+}
```
Comment (Ihor-Karpyn) [inline src/styles/blocks/button-buy.scss:20]:
```suggestion
.button-buy {
  @extend %flex-center;
  border: 1px solid $blue-accent;
  border-radius: 5px;
  background-color: $blue-accent;
  color: white;
  
  &:hover {
    background-color: #fff;
    color: $blue-accent;
  }

  &__link {
    line-height: 16px;
    font-weight: $weight-bold;
    font-size: $m-text;
    text-decoration: none;
    color: inherit;
  }
}
```

File: src/styles/utils/_extends.scss [hunk 2]
```diff
@@ -0,0 +1,20 @@
+%flex-between {
+  display: flex;
+  justify-content: space-between;
+}
+
+%flex-center {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
```
Comment (Ihor-Karpyn) [inline src/styles/utils/_extends.scss:10]:
```suggestion

```

File: src/styles/utils/_variables.scss [hunk 3]
```diff
@@ -0,0 +1,27 @@
+$nav-height: 60px;
+$nav-horizontal-p: 50px;
```
Comment (Ihor-Karpyn) [inline src/styles/utils/_variables.scss:2]:
```suggestion

```

File: src/styles/utils/_variables.scss [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+$card-width: 200px;
+$cards-row: 4;
+$cards-gap: 48px;
+$cards-vertical-margin: 50px;
```
Comment (Ihor-Karpyn) [inline src/styles/utils/_variables.scss:7]:
```suggestion
```

You use this only one time. Why you need var for this one?

File: src/styles/utils/_variables.scss [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+$stars-size: 16px;
```
Comment (Ihor-Karpyn) [inline src/styles/utils/_variables.scss:9]:
```suggestion

```

File: src/styles/utils/_variables.scss [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+$buy-height: 40px;
```
Comment (Ihor-Karpyn) [inline src/styles/utils/_variables.scss:11]:
```suggestion
```

File: src/styles/utils/_variables.scss [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+$xs-text: 10px;
+$s-text: 12px;
+$m-text: 14px;
+$l-text: 16px;
+
+$weight-thin: 400;
+$weight-normal: 500;
+$weight-bold: 700;
```
Comment (Ihor-Karpyn) [inline src/styles/utils/_variables.scss:20]:
```suggestion

```

redundant, it is default variables

https://developer.mozilla.org/ru/docs/Web/CSS/font-weight

File: src/styles/utils/_variables.scss [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+
+$line-small: 14px;
+$line-big: 18px;
```
Comment (Ihor-Karpyn) [inline src/styles/utils/_variables.scss:23]:
```suggestion

```

File: src/styles/utils/_variables.scss [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+
+$blue-accent: #00acdc;
+$main-accent: #060b35;
+$secondary-accent: #616070;
```
Comment (Ihor-Karpyn) [inline src/styles/utils/_variables.scss:27]: color ?

**Review by Ihor-Karpyn — CHANGES_REQUESTED** [review]
Nice start, let's resolve some problems

File: src/styles/utils/_variables.scss [hunk 10]
```diff
@@ -0,0 +1,14 @@
...
+$cards-gap: 48px;
+
+$xs-text: 10px;
+$s-text: 12px;
+$m-text: 14px;
+$l-text: 16px;
+
+$color-blue-accent: #00acdc;
+$color-main-accent: #060b35;
+$color-secondary-accent: #616070;
```
Comment (Ihor-Karpyn) [inline src/styles/utils/_variables.scss:14]:
```suggestion
$color-blue: #00acdc;
$color-main: #060b35;
$color-secondary: #616070;
```

**Review by Ihor-Karpyn — APPROVED** [review]
LGTM🔥!!!

**Pull request closed without merging**